simplq/src/styles/homePage.module.scss

```diff
@@ -143,3 +143,110 @@
     font-size: small;
   }
 }
+
+.burger{
+  width: 2rem;
+  height: 2rem;
+  position: fixed;
+  top: .2rem;
+  right: 1rem;
+  z-index: 20;
+  display: none;
+  @media (max-width: 768px) {
+    display: flex;
+    justify-content: space-around;
+    flex-flow: column nowrap;
+  }
+  div {
+    width: 2rem;
+    height: 0.25rem;
+    background-color: #fff;
+    border-radius: .5rem;
+    transform-origin: 1px;
+    transition: all 0.3s linear;
+  
+    &:nth-child(1) {
+      $type: open;
+      @if $type == open {
+        transform: rotate(45deg);
+      } @else{
+          transform: rotate(0);
+      }
+    }
+    &:nth-child(2) {
+      $type: open;
+      @if $type == open {
+        transform: translateX(100%);
+      } @else{
+          transform: translateX(0);
+      }
+    }
+    &:nth-child(3) {
+      $type: open;
+      @if $type == open {
+        transform: rotate(-45deg);
+      } @else{
+          transform: rotate(0);
+      }
+    }
+  }
+}
+
+
+  .menu-list{
+    list-style: none;
+    display: flex;
+    flex-flow: row nowrap;
+    margin: 0;
+    position: fixed;
+    top: 1.2rem;
+    li {
+      &:nth-child(1){
+        padding: 0px 4rem;
+      }
+    }
+    a{
+      text-decoration: none;
+      color: grey;
+      cursor: auto;
+    }
+    @media (max-width: 768px) {
+      flex-flow: column nowrap;
+      background-color: #0D2538;
+      position: fixed;
+      top: 0;
+      $type: open;
+      @if $type == open {
+        transform: translateX(0);
+      } @else{
+          transform: translateX(100%);
+      }
+      right: 0;
+      height: 100vh;
+      width: 25rem;
+      padding-top: 3.5rem;
+      transition: transform 0.3s ease-in-out;
+      li {
+        color: #fff;
+      }
+  }
+}
+
+
+.navbar{
+    width: 100%;
```

Comment:
![image](https://user-images.githubusercontent.com/11256376/94037618-2cd85500-fde3-11ea-8db9-baf4487e1eaa.png)

On scrolling down, the div with textbox and buttons are coming above the navbar. Adding an `z-index` of 5 here might take care of this. Can you take a look, and then scroll all the way down to see if all other elements are coming below the navbar as expected

Reply:
Do these together:
```
z-index: 99;
background-color: #fff;

Reply:
This issue is still present. I see it here https://arkumari.dpmhbfuiok7f8.amplifyapp.com/ the images on the create queue and join queue sections while scrolling to the bottom. The navbar also blocks the Fork on Github ribbon. I think increasing the z-index for the navbar to something above that for the images and then increasing the z-index for the ribbon to an even higher value should fix this. Could you try this out?

This will help you understand how z-index works: https://www.freecodecamp.org/news/4-reasons-your-z-index-isnt-working-and-how-to-fix-it-coder-coder-6bc05f103e6c/

Reply:
Let's forget about the fork ribbon. I though we were planning on removing that? @maaverik If so, we can remove it from code too.

We could keep a button in the navbar? https://github.com/SimplQ/simplQ-frontend/issues/252

Reply:
Yes, that's right. Don't mind what happens to the fork ribbon.

Reply:
I have already fixed this scrolling problem, I don't know why it is not reflecting in this https://arkumari.dpmhbfuiok7f8.amplifyapp.com/.

z-index: 5; is working properly.
![Screenshot from 2020-09-25 19-40-20](https://user-images.githubusercontent.com/48750169/94277280-fc212880-ff66-11ea-96d6-8c2c8faf4df2.png)

Reply:
Okay, I ran locally and checked. Looks like these issues are solved. We're good to merge, right @daltonfury42?

Reply:
My bad, the preview branches for PRs from forks are to be manually updated, which I forgot to do.